app/controllers/collections_controller.rb

```diff
@@ -82,7 +82,9 @@ def wait
   def works
     authorize! @collection
 
-    @works = @collection.works.order(:title).page(params[:page])
+    @works = authorized_scope(@collection.works, as: :collection, type: :relation,
```

Comment:
This is where the scope is set for the collection works.

Reply:
@justinlittman Did you find you needed to pass the `type: :relation` kwargs? For some reason I thought that was the default value but I could be mistaken.

Reply:
If I made both of your suggested changes, this worked. Thanks!